types/src/service.rs

```diff
@@ -1,10 +1,7 @@
-//! Database [`ReadRequest`]s, [`WriteRequest`]s, and [`Response`]s.
-//!
-//! See [`cuprate_database`](https://github.com/Cuprate/cuprate/blob/00c3692eac6b2669e74cfd8c9b41c7e704c779ad/database/src/service/mod.rs#L1-L59)'s
-//! `service` module for more usage/documentation.
+//! Database [`BCReadRequest`]s, [`BCWriteRequest`]s, and [`BCResponse`]s.
 //!
 //! Tests that assert particular requests lead to particular
-//! responses are also tested in `cuprate_database`.
+//! responses are also tested in Cuprate's blockchain database crate.
```

Comment:
```suggestion
//! responses are also tested in Cuprate's blockchain database crate (`cuprate_blockchain`).
```

Reply:
I don't want to name this explicitly just in case it changes in the future and we forget to update this